tests/migrations/test_autodetector.py

```diff
@@ -657,6 +674,14 @@ def test_alter_field(self):
         self.assertOperationTypes(changes, 'testapp', 0, ["AlterField"])
         self.assertOperationAttributes(changes, "testapp", 0, 0, name="name", preserve_default=True)
 
+    def test_partial_doesnt_detect_changes_when_unchanged(self):
+        """#26475 - Tests functools.partial serialisation always showing changes."""
```

Comment:
I'd inline the ModelStates since they aren't reused, ala http://dpaste.com/0TKY2D5 (though if you add more tests, maybe they will be).